#!/usr/bin/env python
PACKAGE = "mppi_omav_interaction"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

######################
### General Config ###
######################

gen.add("mppi_publish", bool_t, 0, "whether mppi publishes trajectories", False)

######################
### Goal Positions ###
######################

gen.add("goal_pos_x", double_t, 0, "Goal Position x", 0.0, -5.0, 5.0)
gen.add("goal_pos_y", double_t, 0, "Goal Position y", 0.0, -5.0, 5.0)
gen.add("goal_pos_z", double_t, 0, "Goal Position z", 0.5, 0.0, 2.0)

gen.add("reset_object", bool_t, 0, "reset object", False)

exit(gen.generate(PACKAGE, "omav_interaction_goal", "MPPIOmavGoal"))
